elf/cmdline.cc

```diff
@@ -9,7 +9,7 @@
 #include <unordered_set>
 
 #ifdef _WIN32
-# define _isatty isatty
+# define isatty _isatty
```

Comment:
Ditto

Reply:
These, and the above for `common/main.cc`, are to prevent some of the following types of warning:

> warning: 'write' is deprecated: The POSIX name for this item is deprecated. Instead, use the ISO C and C++ conformant name: _write. See online help for details. [-Wdeprecated-declarations]

I noticed the existing use of such definitions in some of the source, although they appeared to be the wrong way around in some instances, like the above.